### PR TITLE
Add styling to modal. Add styling to quotes. Add gif and polished readme

### PR DESCRIPTION
**Pull request by ErikPlachta**

### CODE REVIEW

**Comment by ErikPlachta** [comment]
@barrantesc Completed and ready for final review.

File: assets/css/style.css [hunk 1]
```diff
@@ -108,6 +108,17 @@ a {
...
 .modal {
   flex-direction: unset;
+  text-align: center;
+}
+
+.modal-card-foot {
+  align-items: center;
+}
+
+#makeMeHappy{
```
Comment (ErikPlachta) [inline assets/css/style.css:118]: Did this to make font bigger

File: assets/css/style.css [hunk 2]
```diff
@@ -108,6 +108,17 @@ a {
 
 .modal {
   flex-direction: unset;
+  text-align: center;
```
Comment (ErikPlachta) [inline assets/css/style.css:111]: Making sure everything is center

File: assets/css/style.css [hunk 3]
```diff
@@ -271,29 +283,29 @@ nav span {
 
 /* The container holding all cats */
 .catCards{
-  width: 95%;
-  margin: 1rem auto;
+  width: 100%;
```
Comment (ErikPlachta) [inline assets/css/style.css:286]: Made 100% vs 95, so goes edge-to-edge

File: assets/css/style.css [hunk 4]
```diff
@@ -331,7 +343,8 @@ nav span {
   text-align: center;
   margin-bottom: auto;
   color: white;
-  padding: 1rem 0;
+  padding: 1rem;
+  font-style: bold;
 }
```
Comment (ErikPlachta) [inline assets/css/style.css:348]: Evenly spaced around quotes and bold to be easier tor ead

File: assets/css/style.css [hunk 5]
```diff
@@ -364,7 +377,7 @@ footer {
 
 @media screen and (max-width: 1500px) {
   .carCard {
-    flex: 50%;
+    flex: 40%;
   }
```
Comment (ErikPlachta) [inline assets/css/style.css:381]: Making 40% means 2 x columns with our design.

File: assets/css/style.css [hunk 6]
```diff
@@ -401,17 +416,7 @@ footer {
...
-    }
-    
-    header {
-      border-radius: 2rem 2rem 0 0;
-    }
-
-    footer {
-      border-radius: 0 0 2rem 2rem;
-    }
+
```
Comment (ErikPlachta) [inline assets/css/style.css:419]: Did not need this styling any longer

File: assets/css/style.css [hunk 7]
```diff
@@ -420,5 +425,14 @@ footer {
...
+@media screen and (max-width: 410px) {
+
+  #hamburger,
+  #appName{
+    font-size: 2rem;
+  }
+
+}
+
 /* MEDIA SPECIFIC --> END  */
```
Comment (ErikPlachta) [inline assets/css/style.css:437]: Made smaller so fit properly when scaling down

File: index.html [hunk 8]
```diff
@@ -34,25 +34,24 @@
 
 <body>
 
-  <section> 
-    
+  <section class="morebutton_Modal"> 
     <div id="modal" class="modal">
```
Comment (ErikPlachta) [inline index.html:38]: Added section class name so was more explicit. Not needed atm, but future use

File: index.html [hunk 9]
```diff
@@ -34,25 +34,24 @@
...
         <div class="modal-background"></div> 
         <div class="modal-card">
           <header class="modal-card-head">
-            <p class="modal-card-title"> Purrfect-Friends </p>
+            <p class="modal-card-title">Purrfect-Friends</p>
           </header>
```
Comment (ErikPlachta) [inline index.html:43]: Removed spacing around to prevent styling issues in future

File: index.html [hunk 10]
```diff
@@ -34,25 +34,24 @@
...
           <section class="modal-card-body">
             <!-- Content ... -->
             <h1> Are you sad/depressed? </h1>
             <h2> Come see some cute cats so you can be happy again :D </h2>
           </section>
           <footer class="modal-card-foot">
-            <button id="makeMeHappy" class="button is-success"> Make me happy </button>
+            <button id="makeMeHappy" class="button is-success"> Make Me Happy!</button>
           </footer>
```
Comment (ErikPlachta) [inline index.html:51]: capitalized because button

**Review by barrantesc — APPROVED** [review]

**Review by barrantesc — APPROVED** [review]

**Review by barrantesc — APPROVED** [review]

**Pull request merged**